### PR TITLE
Add stdlogger to exports, add example

### PR DESCRIPTION
**Pull request by geek**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -319,5 +319,6 @@ module.exports = {
     requirements: {
         hapi: '>=18.0.0'
     },
-    pkg: require('../package.json')
+    pkg: require('../package.json'),
+    Stdlogger
```
Comment (devinivy) [inline lib/index.js]: I think the standard way to export something in addition to a plugin is to structure it like this:
```js
exports.plugin = { register, requirements, pkg };

exports.Stdlogger = Stdlogger;
```

hapi knows to check for a `plugin` property like this 👍

File: example/externalLogger.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+'use strict';
```
Comment (devinivy) [inline example/externalLogger.js:1]: Fun example!  I dig it.

File: lib/index.js (hunk 1 above)
Comment (geek) [reply lib/index.js]: Updated

**Review by Nargonath — APPROVED** [review]
Interesting example, TIL. 😁

**Pull request merged**